webpack.config.ts

```diff
@@ -38,7 +39,7 @@ const webpackConfig: any = {
     readline: 'empty',
   },
   module: {
-    noParse: [/\.json$/, /anchor-js/],
+    noParse: [/anchor-js/],
```

Comment:
Without this change, resulting package does not contain files like `package.json`, `Accordion.info.json`...